<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="layout_connect">
        <item>
         <widget class="QLabel" name="label_info_ip">
          <property name="text">
           <string>Please, write ip of the server:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_server_ip">
          <property name="text">
           <string>192.168.199.3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_connect">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_info">
        <item>
         <widget class="QLabel" name="label_info_status">
          <property name="text">
           <string>Status of the server:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_status">
          <property name="text">
           <string>Please, push the button &quot;Connect&quot;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_auth">
        <item>
         <widget class="QLabel" name="label_info_nick">
          <property name="text">
           <string>Please, write your nickname:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_nick"/>
        </item>
        <item>
         <widget class="QPushButton" name="button_auth">
          <property name="text">
           <string>Check and Play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_auth">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>369</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
